symphony/lib/toolkit/class.field.php

```diff
@@ -1538,10 +1538,25 @@ public function buildDSRetrievalSQL($data, &$joins, &$where, $andOperation = fal
     }
 
     /**
+     * Determine if the requested $order is random.
+     *
+     * @since Symphony 2.7.0
+     * @param string $order
+     *  the sorting direction.
+     * @return boolean
+     *  true if the $order is either 'rand' or 'random'
+     */
+    public static function isRandomOrder($order)
```

Comment:
Why static and public?

* All field types extends the class `Field` and so there is no need for a static or public method.
* Maybe I am wrong, but an extension developer also extends the class `Field`. This is another reason against static and public.